pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -304,4 +309,6 @@ def save_checkpoint(self, checkpoint: Dict[str, Any], filepath: str) -> None:
             filepath: write-target file's path
         """
         # Todo: TypeError: 'mappingproxy' object does not support item assignment
+        if _OMEGACONF_AVAILABLE:
+            checkpoint = apply_to_collection(checkpoint, (DictConfig, ListConfig), OmegaConf.to_container)
```

Comment:
looks like this is something that should go to the trainer who calls the accelerator.save_checkpoint(), or not?
Hard to believe this is specific to just tpu spawn.

Reply:
Opened an issue there: https://github.com/pytorch/xla/issues/2846. The issue is caused by this line in torch-xla repo: https://github.com/pytorch/xla/blob/master/torch_xla/utils/utils.py#L201